tests/core/mempool/test_mempool_manager.py

```diff
@@ -346,10 +357,14 @@ async def get_coin_record(coin_id: bytes32) -> Optional[CoinRecord]:
 
     mempool_manager = await instantiate_mempool_manager(get_coin_record)
     conditions = [[ConditionOpcode.CREATE_COIN, IDENTITY_PUZZLE_HASH, amount]]
-    result = await generate_and_add_spendbundle(mempool_manager, conditions)
-    _, status, error = result[2]
-    assert status == expected_status
-    assert error == expected_error
+    try:
+        result = await generate_and_add_spendbundle(mempool_manager, conditions)
```

Comment:
the reason this can throw now is because some failures (specifically `MINTING_COIN` and `RESERVE_FEE`) happen earlier, in prevalidate mempool item. Failures in that funcion throw, whereas failures in the main validation function return an error code.